### PR TITLE
Add support for server alive messages to client

### LINKED ISSUE #10: Clients don't quit when SSHD connection is silently dropped
It looks like there are some circumstances where the SSHD connection between a Zenith client and server can be silently dropped in some way at the server end but the client doesn't exit, because no traffic flowing down the tunnel is a perfectly valid state for the tunnel. This seems to happen especially when using OpenStack Octavia load balancers.

Manually restarting the client solves the problem, but it would be better if the client could automatically detect this situation and restart itself.

The SSH client has a parameter `ServerAliveInterval` that can be set that causes the client to send messages to check that there is still an SSHD server at the other end of the connection. We should set it.

### PR DESCRIPTION
**Pull request by mkjpryor**
Fixes #10

### CODE REVIEW

File: client/zenith/client/config.py [hunk 1]
```diff
@@ -79,6 +79,10 @@ class Config:
     forward_to_host: str = "localhost"
     #: The port to forward tunnel traffic to
     forward_to_port: int = 8000
+    #: The period after which a server alive message will be sent if no data is received
+    server_alive_period: conint(gt = 0) = 10
+    #: The number of server alive messages that can fail before the tunnel is terminated
+    server_alive_failures: conint(gt = 0) = 3
```
Comment (JohnGarbutt) [inline client/zenith/client/config.py:85]: Is 3 failures too many to wait? does that mean waiting 40 seconds to timeout?

Comment (mkjpryor) [reply client/zenith/client/config.py:85]: This means 30s. I think this is reasonable TBH but happy to reduce.

Comment (mkjpryor) [reply client/zenith/client/config.py:85]: Note that if the connection is properly terminated by either end, this doesn't apply - the process exits immediately. This is just to cover the case where the server end is silently dropped with no notification to the client.

Comment (JohnGarbutt) [reply client/zenith/client/config.py:85]: Yeah, I was thinking it means wiat 10 seconds, then fail, and fail on the fourth. I was thinking only allowing one failure seems OK, but this is all better than forever!

**Pull request merged**